letsencrypt/cli.py

```diff
@@ -862,9 +888,10 @@ def _reconstitute(config, full_path):
                        "invalid. Skipping.", full_path)
         return None
 
-    setattr(config.namespace, "domains", domains)
-    return renewal_candidate
+    if not _set_by_cli("domains"):
```

Comment:
since domains defaults to [] and the new add passes on it, could this be replaced with if not getattr(config.namespace, "domains")
though I could see why you might not want to do that for uniformity